### PR TITLE
completed react-function-components - Jay Ghil

### PR DESCRIPTION
**Pull request by jghil**
image

### CODE REVIEW

File: react-function-components/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+
+root.render(CustomButton());
```
Comment (scbowler) [inline react-function-components/src/index.jsx]: We never call a component directly. Instead we use JSX:
```JSX
root.render(<CustomButton />);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Heading in the right direction, but one update needed.

File: react-function-components/src/index.jsx (hunk 1 above)
Comment (jghil) [reply react-function-components/src/index.jsx]: gotcha! used JSX instead!

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**